cypress/integration/userShipping.spec.js

```diff
@@ -0,0 +1,8 @@
+describe("user/shipping", () => {
+  describe("unauthenticated", () => {
+    it("redirects to the login page", () => {
+      cy.visit("user/shipping")
+      cy.contains("Login to Artsy")
```

Comment:
Nice catch by the cypress tests! Looks like the handling of unauthenticated requests for `/user/shipping` is inconsistent with `/user/payments`. The shipping route will 404 instead of redirecting to the log in page. Is this something we want to fix?

![Screen Shot 2021-07-22 at 12 52 40 PM](https://user-images.githubusercontent.com/796573/126677919-b7325402-3856-4714-9fc1-039843d90137.png)